projects/angular-cld/src/lib/version.ts

```diff
@@ -0,0 +1 @@
+export let APP_VERSION = '1.3.2';
```

Comment:
I suggest the default value will be something outstanding, like 0.0.0 or 9.9.9

Reply:
shouldn't it be the current version?

Reply:
It's not going to ever change (as you only update during the release automation and never commit back to the repo), so you better have something that will stand-out if something was not created right